releng/tools/org.openjdk.jmc.util.releasenotes/notes/9.0/notes.xml

```diff
@@ -0,0 +1,261 @@
+            <image>general/jmc.png</image>
+        </note>
+		<note version="9.0">
+			<title>Eclipse 4.30 support</title>
+			<description>
+				The Mission Control client is now built to run optimally
+				on Eclipse 2023-12 and later. To install JDK Mission Control into
+				Eclipse, go to the update site (Help | Install New Software...). The
+				URL to the update site will be vendor specific, and some vendors
+				will instead provide an archive with the update site.
+			</description>
+			<image>general/eclipse.png</image>
+		</note>
+		<note version="9.0">
+			<title>Support for Linux/aarch64</title>
+			<description>
+				JMC 9 is now built for Linux aarch64.
+			</description>
+			<image>general/eclipse.png</image>
+		</note>
+		<note version="9.0">
+			<title>Minor bugfixes and improvements</title>
+			<description>
+				There are 118 fixes and improvements in this release.
+				Check out the JMC 9.0 Result Dashboard
+				(https://bugs.openjdk.org/secure/Dashboard.jspa?selectPageId=21800)
+				for more information.
+			</description>
+			<image>general/issues.png</image>
+		</note>
+        <note version="9.0">
+            <title>Add user configuration for local JVM refresh interval</title>
+            <description>
+				Previously the JVM Browser checked every 5000 ms [0] for new JVMs. This can now be configured.
+            </description>
+            <image>general/refreshinterval.png</image>
+        </note>
+    </component>
+    <component>
+        <name>Core</name>
+        <!-- Core library enhancements for JMC 9 -->
+        <note version="9.0">
+            <title>Better JFR parser performance</title>
+            <description>
+            	Multiple efforts have been made to reduce allocations in the JMC parser, including:
+            	reduced allocation of Doubles, reduced allocation rate in ParserStats.
+            	Also, when duration events aren't ordered by their end time (e.g. events which stack so that the last 
+            	event finishes first, or file reads with overlaps) `DisjointBuilder.add` can be slow because of 
+            	the linear search for the lane, and then a linear time reordering. This has been improved with 
+            	a binary search.
+            </description>
+            <image>jfr/jfrperformance.png</image>
+        </note>        
+        <note version="9.0">
+            <title>Support checkpoint event sizes beyond u4 limit</title>
+            <description>
+            	The JMC JFR parser now support checkpoint event sizes beyond the u4 limit.
+            </description>
+            <image>jfr/checkpointsize.png</image>
+        </note>        
+        <note version="9.0">
+            <title>Move non-Eclipse dependant classes from org.openjdk.jmc.ui.common to org.openjdk.jmc.common</title>
+            <description>
+            	There are a number of classes currently in jmc.ui.common that would be a great asset to the core distribution (and the 
```

Comment:
Since the refactoring has happened, I think the first part of this sentence should probably be put in the past tense, something like:

There were a number of classes previously in jmc.ui.common that are a great asset to the core distribution (and the third-party applications that consume jmc core libraries), and these classes now live in jmc.common.